PyISY/Connection.py

```diff
@@ -196,6 +196,21 @@ def nodeFastOn(self, nid):
         response = self.request(req_url)
         return response
 
+    def nodeSlowFadeUp(self, nid):
```

Comment:
What's the thinking behind calling it "Slow" fade up and down? If this matches some ISY documentation or something then that's fine, but I thought this was generally just called Fade Up.

Reply:
@boojew any thoughts on this one?

Reply:
FWIW: in the WSDK it's just Fade, not Slow Fade

| Command | Name |
| - | - |
| FDDOWN | Fade Down |
| FDSTOP | Fade Stop |
| FDUP | Fade Up |